src/index.html

```diff
@@ -8,5 +8,39 @@
   </head>
   <body>
     <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-fri calendar--month-length--28">
+      <div class="calendar__day calendar__day--1"></div>
+      <div class="calendar__day calendar__day--2"></div>
+      <div class="calendar__day calendar__day--3"></div>
```

Comment:
```suggestion
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

Each cell should have only calendar__day